ivy/functional/ivy/linear_algebra.py

```diff
@@ -183,13 +183,16 @@ def cross(
          an array containing the cross products. The returned array must have a data
          type determined by Type Promotion Rules.
 
-    This method conforms to the `Array API Standard
-    <https://data-apis.org/array-api/latest/>`_. This docstring is an extension of the
```

Comment:
"This docstring is an extension of the..." should not have been removed. Check [here](https://lets-unify.ai/ivy/deep_dive/12_docstrings.html).